src/region/RegionAnnotations.tsx

```diff
@@ -75,6 +72,14 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
         createRegion({ ...staged, message });
     };
 
+    resetStaged = (): void => {
+        const { setMessage, setStaged, setStatus } = this.props;
+
+        setMessage('');
```

Comment:
Do we need to reset the cursor and error properties?

Reply:
> Do we need to reset the cursor and error properties?

The cursor will be reset in ReplyField iirc. There is no setError reducer currently. I can do it in the error handling ticket.